f/fancytree/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright 2008-2021 Martin Wendt,
```

Comment:
This is not the package license, but the script license, which is Apache 2.0, please update